source/databricks/calculation_engine/package/calculation/output/results/schemas/amounts_per_charge_schema.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyspark.sql.types import (
+    ArrayType,
+    BooleanType,
+    DecimalType,
+    StringType,
+    StructField,
+    StructType,
+    TimestampType,
+)
```

Comment:
I think our common practice has turned towards:
```suggestion
import pyspark.sql.types as t
```

Reply:
In some cases I find it more disturbing having to add "t." than the value it adds. LIke in this isolated schema defintion. It adds a tiny bit more effort when typing and I'm not sure about what I get. Anyways, I have followed your/our recommendation. Just wanted to share my thougt.

Reply:
Perhaps we should have a brief team talk about it and update our documentation on our best practice (I don't think we have such a thing, but I believe we should have).